# This message is used to set the onohat status led

bool value
---
bool result